packages/core/content-type-builder/admin/src/pages/ListView/ListView.tsx

```diff
@@ -19,6 +20,10 @@ import { LinkToCMSettingsView } from './LinkToCMSettingsView';
 
 /* eslint-disable indent */
 
+const LayoutsHeaderCustom = styled(Layouts.Header)`
+  overflow-wrap: anywhere;
```

Comment:
I wonder why anywhere and not break-word? Looking at the MDN docs it looks like it could affect margin/padding sizes

Reply:
Because we want to break long title with just one word, the break-word in this case doesn't work